src/views/MediaChannels/components/MoreInfoDrawer.tsx

```diff
@@ -1,15 +1,88 @@
-import { Drawer, makeStyles, Theme, Typography } from "@material-ui/core";
+import {
+  Box,
+  Button,
+  Chip,
+  Drawer,
+  Grid,
+  IconButton,
+  Link,
+  makeStyles,
+  Theme,
+  Tooltip,
+  Typography,
+} from "@material-ui/core";
+import AccessTimeIcon from "@material-ui/icons/AccessTime";
+import CloseIcon from "@material-ui/icons/Close";
+import CreditCardIcon from "@material-ui/icons/CreditCard";
+import DateRangeIcon from "@material-ui/icons/DateRange";
+import InfoIcon from "@material-ui/icons/Info";
 import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { isValidHttpUrl } from "@temp/core/utils";
+import { messages } from "@temp/views/MediaChannels/messages";
 
 const useStyles = makeStyles<Theme>(theme => ({
   root: {
-    width: 400,
+    width: "min(500px, 100vw)",
+  },
+  content: {
+    padding: theme.spacing(6, 2, 2, 2),
+    overflowX: "hidden",
+    overflowY: "auto",
+  },
+  paragraph: {
+    marginBottom: theme.spacing(4),
   },
-  moreInfoContent: {
-    padding: theme.spacing(4),
+  chips: {
+    margin: theme.spacing(-0.5, -0.5, 3.5, -0.5),
+    display: "flex",
+    flexWrap: "wrap",
+    "& > *": {
+      margin: theme.spacing(0.5),
+    },
   },
-  moreInfologo: {
+  close: {
+    position: "absolute",
+    textAlign: "end",
+    top: 0,
     width: "100%",
+    background:
+      "linear-gradient(180deg, white 0%, white 75%, transparent 100%)",
```

Comment:
is this (and other similar styles) theme-specific? shouldn't it be extracted?

Reply:
Only if the background will not be white. But for now I just removed the gradient. That way it is always correct.